packages/storybook/src/stories/grid-fluid.stories.tsx

```diff
@@ -0,0 +1,126 @@
+import * as React from 'react';
+import { XGrid } from '@material-ui/x-grid';
+import { withKnobs } from '@storybook/addon-knobs';
+import { withA11y } from '@storybook/addon-a11y';
+import '../style/grid-stories.css';
+
+export default {
+  title: 'X-Grid Tests/Fluid Column Width',
```

Comment:
Could we embed them in `grid-columns.stories.ts` please

Reply:
done